### ISSUE TITLE
Win-mix-real.. Oculus Medium

### ISSUE THREAD

**Issue by cmarkme** [LibreVR/Revive#934]
Hey, Oculus Medium Crashes, every time I use Revive with my Lenovo Headset WMR.. Everything else in the Revive Department runs fine.. Is there anything that can be done?? :-) I'll paste my Log below

[Oculus_Medium.log](https://github.com/LibreVR/Revive/files/1767403/Oculus_Medium.log)

**Comment by Armada651** [comment]
You need to switch to the SteamVR beta and turn off your Steam Overlay:

![medium-overlay](https://user-images.githubusercontent.com/2028870/36792739-af230cc0-1c9b-11e8-8648-67846b4143c1.png)

**Comment by cmarkme** [comment]
Hey that was quick.. im in steamVR Beta.. but ill check the Overlay thing give me a Momento :-) Thanks X

**Comment by cmarkme** [comment]
No Joy.. still comes up with the same Error.. 'Oculus Medium has Crashed'  I'm not sure whats happening ?? :-)

**Comment by Armada651** [comment]
Then you'll have to wait until the release of Remixed for WMR, only the Vive is officially supported in Revive.

**Comment by cmarkme** [comment]
True.. thanks for your Extremely fast reply my Friend .. and the information is new to me about 'Remixed' so youve got me all a Flutter  :-)
